pkg/dssproto/README.md

```diff
@@ -1,16 +0,0 @@
-All of these files are generated using
```

Comment:
Let's move this README to /pkg/api and update it to explain how the files in that folder's subfolders are generated -- we don't want to regress on amount of documentation.

Reply:
done